### PR TITLE
Adding the remit-hello-interval option in the multicast command

### PR DESCRIPTION
**Pull request by fernandoGietz**
Create a little feature to add the possibility to config the remit-hello-interval in the udp-sender commands.
NOTE: require alter the nfsGroupMembers table in the database

### CODE REVIEW

File: packages/web/lib/pages/storagemanagementpage.class.php [hunk 1]
```diff
@@ -411,12 +412,19 @@ public function addStorageNode()
             . '" autocomplete="off" class="form-control"/>'
             . '</div>',
             '<label for="bitrate">'
-            . _('Bitrate')
+            . _('Bitrates')
```
Comment (Sebastian-Roth) [inline packages/web/lib/pages/storagemanagementpage.class.php]: @fernandoGietz Did you change that label by intention?

Comment (fernandoGietz) [reply packages/web/lib/pages/storagemanagementpage.class.php]: No, a fat finger??

**Comment by Sebastian-Roth** [comment]
Looks fine from my point of view. The change of schema.php will be a bit of an issue when merging into working-1.6 (@mastacontrola) but there is no way around it.

**Comment by mastacontrola** [comment]
I think we can do the schema update relatively simply.

For working-1.6 I just add:
```
IF NOT EXISTS( SELECT NULL
            FROM INFORMATION_SCHEMA.COLUMNS
           WHERE table_name = 'nfsGroupMembers'
             AND table_schema = 'fog'
             AND column_name = 'ngmHelloInterval')  THEN
  ALTER TABLE `fog`.`nfsGroupMembers` ADD `ngmHelloInterval` VARCHAR(8) AFTER `ngmMaxBitrate`;
END IF;
```

**Comment by mastacontrola** [comment]
Alright added this to working-1.6 with a testable approach. If the column doesn't already exist, create it. Otherwise just make it a null schema entry point.

**Pull request merged**